pkg/server/test/connected_objects.go

```diff
@@ -1200,7 +1266,7 @@ func ConnectedObjectsTest(t *testing.T, ds storage.OpenFGADatastore) {
 				done <- struct{}{}
 			}()
 
-			timeoutCtx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+			timeoutCtx, cancel := context.WithTimeout(context.Background(), 10*time.Hour)
```

Comment:
This was changed in a previous commit and i'm not sure why. @jon-whit @adriantam

Reply:
Oh, that was probably me and I committed it accidentally. I change that when debugging locally so my request doesn't time out while I debug.